### DISCUSSION TITLE
suggestion for install command (typo prevents install)

### DISCUSSION THREAD

**Discussion by ejonesss** [Homebrew/discussions#94] [Everyday usage]
$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

causes command not found error and after a google search i found i had to  use

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

it seems as the $ is the problem

you may want to update the front page as others will have the same problem

i am on catalina on a vmware instance and could not install due to typo

**Comment by SMillerDev** [comment]
This is a standard for representing commands that you have to input in a terminal, they are prefixed by `$`. You can press the copy button to copy without this character.

**Comment by EricFromCanada** [comment]
Updated in https://github.com/Homebrew/brew.sh/pull/536.
